### ISSUE TITLE
Comment replies don't include MoreChildren

### ISSUE THREAD

**Issue by sirkris** [sirkris/Reddit.NET#94]
The issue is in the Things.Comment object:

```c#
        [JsonProperty("replies")]
        public CommentContainer Replies { get; set; }
```

As you can see, the Replies property is defined as a CommentContainer.  Unfortunately, I've just realized that this property can also be a "more"-type listing, which I hadn't observed before because apparently it only exists in deeply-threaded comment replies, so the library doesn't even have a Thing for that right now.  I'll need to add one.

Here's an example of what we're looking at:

```json
[
   {
      kind: "Listing",
      data: {
         modhash: "fdroq5szue77292449297c2f1006fa25f44bcc75be8c22592a",
         dist: 1,
         children: [...],
         after: null,
         before: null
      }
   },
   {
      kind: "Listing",
      data: {
         modhash: "fdroq5szue77292449297c2f1006fa25f44bcc75be8c22592a",
         dist: null,
         children: [
            {
               kind: "t1",
               data: {
                  total_awards_received: 0,
                  approved_at_utc: null,
                  ups: 978,
                  awarders: [ ],
                  mod_reason_by: null,
                  banned_by: null,
                  removal_reason: null,
                  link_id: "t3_2lt3d0",
                  author_flair_template_id: null,
                  likes: null,
                  no_follow: false,
                  replies: {
                     kind: "Listing",
                     data: {
                     modhash: "fdroq5szue77292449297c2f1006fa25f44bcc75be8c22592a",
                     dist: null,
                     children: [
                        {
                           kind: "t1",
                           data: {
                              total_awards_received: 0,
                              approved_at_utc: null,
                              ups: 441,
                              awarders: [ ],
                              mod_reason_by: null,
                              banned_by: null,
                              removal_reason: null,
                              link_id: "t3_2lt3d0",
                              author_flair_template_id: null,
                              likes: null,
                              no_follow: false,
                              replies: {
                                 kind: "Listing",
                                 data: {
                                    modhash: "fdroq5szue77292449297c2f1006fa25f44bcc75be8c22592a",
                                    dist: null,
                                    children: [
                                       {
                                          kind: "t1",
                                          data: {
                                          total_awards_received: 0,
                                          approved_at_utc: null,
                                          ups: 48,
                                          awarders: [ ],
                                          mod_reason_by: null,
                                          banned_by: null,
                                          author_flair_type: "text",
                                          removal_reason: null,
                                          link_id: "t3_2lt3d0",
                                          author_flair_template_id: null,
                                          likes: null,
                                          replies: {
                                             kind: "Listing",
                                             data: {
                                             modhash: "fdroq5szue77292449297c2f1006fa25f44bcc75be8c22592a",
                                             dist: null,
                                             children: [
                                             {
                                                kind: "more",
                                                data: {
                                                   count: 3,
                                                   name: "t1_clyelyg",
                                                   id: "clyelyg",
                                                   parent_id: "t1_clyaevo",
                                                   depth: 3,
                                                   children: [
                                                      "clyelyg",
                                                      "clyiysi"
                                                      ]
                                                   }
                                                }
                                                ],
                                                after: null,
                                                before: null
                                             }
                                          },
```

URL:  https://www.reddit.com/r/news/comments/2lt3d0/top_01s_wealth_about_to_overtake_bottom_90_for/.json?limit=500&sort=confidence

As you can see, the replies are all comment containers until a certain depth is reached, then they display the MoreChildren data, instead.  I'll need to do some refactoring to accommodate this.

**Comment by sirkris** [comment]
Related: #73

**Comment by sirkris** [comment]
I'm currently blocked on this by a possible API bug:

https://www.reddit.com/r/redditdev/comments/dqz2jy/api_bug_the_apiinfo_endpoint_does_not_populate/?

**Comment by sirkris** [comment]
Fixed and merged into 1.3.
